### DISCUSSION TITLE
KSADS Parent Survey ADHD Data Missing

### DISCUSSION THREAD

**Discussion by msf51** [now-i-know-my-abcd/docs#53] [Stats & Code Help]
Hi, 

I've just gotten started working on this data and I am trying to select for participants with any diagnosis of ADHD (past, current, full remission, partial remission, etc...). 

I am using the data set mh_p_ksads_ss but I am not getting the number of expected responses for any of these variables. For example, I found a previous publication that used the variable ksads_14_853_p to select for current ADHD diagnosis and reported 1091 participants with this diagnosis. I do not get that many when I select for this variable and even if I add all the ADHD variables on this set together it adds up to about 200 who responded "yes" to any of these questions. I am including the ones that start with ksads2 as well. 

I am working in SAS and imported the data from the .csv file downloaded from the NDA. When I look at those variables in the .csv file it is primarily blank cells and again not as many responses as expected. 

Has anyone else encountered this problem or does anyone have a potential solution? 

Thanks!

**Comment by msf51** [comment]
Update: I reached out to ABCD and they said it is a known issue with the data and they anticipate releasing a patch to fix it by the end of the year.

**Comment by msf51** [comment]
Update: ABCD advised that I calculate my ADHD diagnoses myself as an alternative to using the summary scores data. Has anyone else done this, or does anyone have a resource where I can find the formula I need to do this? 

Thanks!

**Reply by YanaShimana** [reply]
Hi there, I don't have an answer for you. I am commenting in the hopes that you have had better luck finding a solution because I am also looking to work with the ADHD data. ABCD told me the patch with corrected data will be released next month, but I would love to figure something out sooner than that.

I talked to a coworker of mine who has experience with KSADS about how/whether I could make a determination based off the mh_p_ksads_adhd table. She said that because it's semi-structured, not everyone gets asked every question and when she's done it, she's looked at responses and compared to the DSM which would not be manageable for so many people.

Have you possibly received more information or found a solution?

Thanks so much!

**Reply by msf51** [reply]
I did! It was painstaking but I finally got it. 

I had to figure out the order of survey questions. Verify independently for yourself, but I believe it is: 

Screening questions assessing current symptoms (404,409,414, 419) 
--> if 3 or 4 to ANY, full CURRENT questionnaire and 405,410,415,520 (assessing past symptoms)
--> if 0, 1, or 2 to ANY, 406,411,416,421 (assessing past symptoms)
--> if yes to ANY (405,406,410,411,415,416,420,421) --> asked 408,413,418,423 --> if yes to ANY, full PAST questionnaire 

Diagnostic criteria: 6/9 inattentive or hyperactive symptoms
- Current: 3 or 4 on 404/409/414 and 2, 3, or 4 on 973-988 
- Past:  "yes" on 408/413/418, 996-1010
- Problem in at least 2 settings (current): 990-994
- Problem in at least 2 settings (past): 1012-1016

They also need to experience symptoms prior to age 12 but I didn't include that in my coding because I was just looking at the baseline cohort for now and they are all under 12 at the time of the questionnaire. 

I was also able to reach out to KSADS to confirm the diagnostic criteria for the KSADS-COMP measure. 

Hope this helps, and good luck!

**Reply by haigui0913** [reply]
Hi! Thanks a lot for the detailed exploration and clarification.

Would you mind sharing the code you used to calculate the ADHD diagnosis? Would  you ended up matching that number (N = 1091) as well.
Appreciate your help!
